cmd/ethrex_replay/src/cli.rs

```diff
@@ -0,0 +1,118 @@
+use clap::{Parser, Subcommand};
+
+use crate::bench::run_and_measure;
+use crate::fetcher::{get_blockdata, or_latest};
+use crate::run::{exec, prove};
+
+pub const VERSION_STRING: &str = env!("CARGO_PKG_VERSION");
+
+#[derive(Parser)]
+#[command(name="Ethrex_replay_cli", author, version=VERSION_STRING, about, long_about = None)]
+pub struct EthrexReplayCLI {
+    #[command(subcommand)]
+    command: EthrexReplayCommand,
+}
+
+#[derive(Subcommand)]
+enum SubcommandExecute {
+    #[clap(about = "Execute a single block.")]
+    Block {
+        block: Option<usize>,
+        #[arg(
+            long,
+            default_value = "http://localhost:8545",
+            env = "RPC_URL",
+            required = true
+        )]
+        rpc_url: String,
+        #[arg(long, required = false)]
+        bench: bool,
+    },
+}
+
+impl SubcommandExecute {
+    pub async fn run(self) -> eyre::Result<()> {
+        match self {
+            SubcommandExecute::Block {
+                block,
+                rpc_url,
+                bench,
+            } => {
+                let block = or_latest(block, &rpc_url).await?;
+                let cache = get_blockdata(rpc_url, block).await?;
+                let body = async {
+                    let gas_used = cache.block.header.gas_used as f64;
+                    let res = exec(cache).await?;
+                    Ok((gas_used, res))
+                };
+                let res = run_and_measure(bench, body).await?;
+                println!("{}", res);
+            }
+        }
+        Ok(())
+    }
+}
+
+#[derive(Subcommand)]
+enum SubcommandProve {
+    #[clap(about = "Proves a single block.")]
+    Block {
+        block: Option<usize>,
+        #[arg(
```

Comment:
it can be useful to add a help message here that specifies that this value can be read from the RPC_URL environment variable or it will take the default value.